includes/xml_parsers/EUtilsAssemblyParser.inc

```diff
@@ -0,0 +1,125 @@
+<?php
+
+class EUtilsAssemblyParser implements EUtilsParserInterface {
+
+
+  public function parse(SimpleXMLElement $xml) {
+
+    $info = [];
```

Comment:
I'd predefine the keys here

```suggestion
    $info = [
      'attributes' => [],
      'name' => '',
      'description' => '',
      'accession' => '',
      'ignored' => []
    ];
```

Reply:
hmmm ok we can do that, but it makes checking the keys exist much more important.

Reply:
This is just so data types are clear. You already define those down in the switch statement. This just makes it easier to read when guessing what data this will return.